purchase_request/demo/purchase_request_demo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
+    <data noupdate="0">
```

Comment:
noupdate="1" into demo datas means that if the users play with datas and you update the implied module then the datas of the users will be safe otherwise reset with the xml content. This could be usefull for demo datas and some others real datas like ir.parameter